editor/components/block-list/style.scss

```diff
@@ -699,3 +699,11 @@
 		padding-top: 6px;
 	}
 }
+
+.editor-block-breadcrumb {
```

Comment:
Aside: I applied an invalid class name per standards; this should be `.editor-block-list__breadcrumb`